php-practice/section5.php

```diff
@@ -320,3 +338,25 @@ public function send() {
  *   'cost' => 3.99
  * ];
  */
+
+class ShoppingCart {
+  private $item;
+  private $shoppingCart;
```

Comment:
This class only needs one property -- an array to hold all of the shopping cart items.

Looks like you're using `$shoppingCart` below, so you can get rid of `$item`.